### PR TITLE
Feature/GULAG-T-16 MOVE: create-react-app to create-next-app

### PR DESCRIPTION
**Pull request by Dimashey**
1) Init typescript next.js 
2) setup eslint, prettier, allias
3) conect redux-saga

### CODE REVIEW

File: front-end/package.json [hunk 1]
```diff
@@ -1,43 +1,20 @@
 {
-  "name": "gulag",
+  "name": "front-endend",
```
Comment (ridisi95) [inline front-end/package.json:2]: I think 1 "end" it is enough ?))

File: front-end/yarn.lock [hunk 2]
```diff
@@ -0,0 +1,4040 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (ridisi95) [inline front-end/yarn.lock:1]: Should we add autogenerated files in git repo?

**Review by ridisi95 — CHANGES_REQUESTED** [review]

File: front-end/yarn.lock (hunk 2 above)
Comment (Dimashey) [reply front-end/yarn.lock:1]: This file will needed when we will deply app

Comment (ridisi95) [reply front-end/yarn.lock:1]: We can use job to generate and compile all the project, why we can't autogenerate it ?

File: front-end/package.json (hunk 1 above)
Comment (Dimashey) [reply front-end/package.json:2]: agree

File: front-end/yarn.lock (hunk 2 above)
Comment (ridisi95) [reply front-end/yarn.lock:1]: I found the answer on [StackOverFlow](https://stackoverflow.com/questions/39990017/should-i-commit-the-yarn-lock-file-and-what-is-it-for). Ok It should be commited)

**Review by ridisi95 — APPROVED** [review]

**Pull request merged**